events_mgmt/src/main/java/com/adobe/event/management/model/RegistrationInputModel.java

```diff
@@ -157,6 +157,12 @@ public Builder addEventsOfInterests(
       return this;
     }
 
+    public Builder addMultipleEventsOfInterests(
```

Comment:
wdyt just reusing the same method name `addEventsOfInterests` here ?

Reply:
makes sense considering the method name is plural and intended for that purpose. ill update the PR